easybuild/easyconfigs/a/ASAP3/ASAP3-3.11.10-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.11.10'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['ASAP3-3.11.10-Respect-CFLAGS-for-EasyBuild.patch']
+checksums = [
+    'da1b9bb8cde76cf9010921dfd765acf8e7488ae60e5e57c62ec390a178c89261',  # asap3-3.11.10.tar.gz
+    # ASAP3-3.11.10-Respect-CFLAGS-for-EasyBuild.patch
+    'a46ebd026a6e923ec0a76e916f35f29730ff03f7bce2f7b4dd8838f3b58e7402',
+]
+
+builddependencies = [
+    ('pkgconfig', '1.5.1', '-python'),
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('ASE', '3.18.0', versionsuffix),
+    ('kim-api', '2.1.2'),
+]
+
+use_pip = False
```

Comment:
@schiotz Just wondering, which issues did you run into with `pip install` that made you use `use_pip = False` here?

Reply:
Asap installs a version of the python executable linked with MPI (called asap-python, needed for parallel simulations only), this does not get installed properly when using pip, since pip assumes it is a script, and tries to read the file and replace the hash-bang in the first line with the right path to the python executable.  But since it is not a script but an executable, it goes wrong.

The main GPAW developer is working on fixing this in gpaw so building a special executable is not necessary, but there still seems to be problems on some combinations of architectures and MPI versions.  Eventually, I am going to borrow his solution for asap, but it does require some work.

Reply:
Thanks a lot for the clarification!